package.json

```diff
@@ -377,6 +377,7 @@
     "@types/jest-axe/axe-core": "4.7.2",
     "esbuild": "0.24.2",
     "eslint": "8.57.0",
+    "@microsoft/api-extractor/typescript": "5.3.3",
```

Comment:
just to double check . the bumped api-extractor uses ts 5.5 right ?

Reply:
no, the latest one uses 5.7.2, it jumps from 5.4.2 to to 5.7.2

Reply:
@hotell no, the latest one uses 5.7.2, it jumps from 5.4.2 to to 5.7.2